iphone/Classes/TiUITabProxy.m

```diff
@@ -54,9 +56,18 @@ - (void)_configure
   [self replaceValue:NUMBOOL(YES) forKey:@"activeIconIsMask" notification:NO];
   [self replaceValue:nil forKey:@"titleColor" notification:NO];
   [self replaceValue:nil forKey:@"activeTitleColor" notification:NO];
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(didChangeTraitCollection:)
+                                               name:kTiTraitCollectionChanged
+                                             object:nil];
   [super _configure];
 }
 
+- (void)didChangeTraitCollection:(NSNotification *)info
+{
+  [self updateTabBarItem];
```

Comment:
Shouldn't the UIImage instances use semantic images already and check automatically?

Reply:
There seems a bug from apple side. See https://stackoverflow.com/questions/57011346/uitabbar-uitabbaritem-and-ios-13s-dark-mode

Reply:
Okay. But `traitCollectionDidChange:` fires for iOS 8+ and also if the device is rotated or font scaling is updated. It's minor but should be guarded in a later release to prevent unnecessary UI work.